### ISSUE TITLE
Notewise selection extension in chords

### ISSUE THREAD

**Issue by Magnus26** [musescore/MuseScore#25028]
### Your idea

I would like to be able to vertically notewise extend the selection of notes in a chord with Shift+Up/Down. As of now, all chord notes get selected when one note in a chord is selected and Shift+Up is pressed.

### Problem to be solved

The impossibility with just the computer keyboard to extend a selection with only one or a few notes in a chord.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by Magnus26** [comment]
> As of now, all chord notes get selected when one note in a chord is selected and Shift+Up is pressed.

Sometimes this does not work at all, like in the bass clef staff of a marimba part, or it also selects the staff above and not only the chord in the staff that is worked on.

**Comment by MarcSabatella** [comment]
That’s because Shift+Up is defined to create a range selection by incorporating the next voice or staff.  Shift plus any cursor movement always creates a range selection; that is its standard function.

**Comment by Magnus26** [comment]
> That’s because Shift+Up is defined to create a range selection by incorporating the next voice or staff. Shift plus any cursor movement always creates a range selection; that is its standard function.

Yes, a range och chord notes is what I would like.

**Comment by MarcSabatella** [comment]
And a range selection *is* what is selected.  I think maybe you are misunderstanding what a range selection is.  It different from a list selection in that by definition, a range includes *everything* from a given point in time to another given point in time across a given range of staves.  So there is no such thing as a range selection that includes some notes from a given chord but not others.

It's not impossible that someday this range vs list distinction might go away or at least get more blurred, but until then the way to have a selection containing some notes of a chord but not others is to beu9ild a list selection - eg, Ctrl+click.

It isn't clear what your actual goal is here, but chances are that whatever you are trying to do is easily accomplished some other way.  I recommend asking for help in the official support forum at musescore.org and attaching your score and clear description of what you are trying to do - what you want to accomplish after you make the selection.  Then we can understand and assist better.

**Comment by Magnus26** [comment]
> And a range selection _is_ what is selected.

Not the range I would like. E.g. Ctrl+Shift+Up would be better for the range that is currently selected by MuseScore Studio with Shift+Up, making possible the range selection I want with Shift+Up.

**Comment by MarcSabatella** [comment]
What I am saying is that is simple not what a range selection is.   As I explained, by definition, it is *everything* from a given start time to a given end time in a given set of staves.  So it’s not possible to add a command to create a type of selection that doesn’t exist.

Again, please use the support forum to explain what you are actually trying to do so we can help you understand how to use the program better.

**Comment by Magnus26** [comment]
> What I am saying is that is simple not what a range selection is. As I explained, by definition, it is _everything_ from a given start time to a given end time in a given set of staves.

From where do you take your definition?

**Comment by oktophonie** [comment]
This exchange is getting silly now. It's clear that the request is for a way to expand (or contract) a list selection of adjacent notes within a chord.

**Comment by Magnus26** [comment]
> It's clear that the request is for a way to expand (or contract) a list selection of adjacent notes within a chord.

Thanks, Simon.

**Comment by MarcSabatella** [comment]
Indeed, that much is clear.  My point is just that Shift+cursor has an important function as it is for building range selections, so it's not appropriate to redefine it.  A separate command would be needed.  But, the underlying problem is deeper than that.  There is no way to build list selections by keyboard, period - whether we are talking about vertically adjacent notes in chords, horizontally adjacent notes, or discontiguous sets of notes or other elements.  This is a very significant accessibility limitation that needs to be solved more generally.  See https://github.com/musescore/MuseScore/issues/20497.

**Comment by jmwmusic** [comment]
If this is completely covered by #20497 and it is generally agreed that no additional keyboard shortcut outside of a way to emulate a Ctrl Click is needed can this be closed?
